web/src/lib/components/layouts/user-page-layout.svelte

```diff
@@ -27,21 +29,23 @@
 >
 	<SideBar />
 
-	<slot name="content">
-		<section class="my-8 mx-4 bg-immich-bg dark:bg-immich-dark-bg">
-			{#if title}
-				<div class="flex justify-between place-items-center dark:text-immich-dark-fg px-4 h-10">
-					<p class="font-medium">{title}</p>
-
-					<slot name="buttons" />
-				</div>
-
-				<div class="my-4">
-					<hr class="dark:border-immich-dark-gray" />
-				</div>
-			{/if}
-
-			<slot />
-		</section>
-	</slot>
+	<ConditionalWrapper condition={!isTimeline} class="overflow-y-auto">
+		<slot name="content">
+			<section class="my-8 mx-4 bg-immich-bg dark:bg-immich-dark-bg">
```

Comment:
Why is this conditional wrapper even needed? The photos page overrides the content slot, so adding `overflow-y-auto` to the section element should work

Reply:
You are right ;)
Done